typeguard/__init__.py

```diff
@@ -26,6 +26,11 @@
 except ImportError:
     Literal = TypedDict = None
 
+try:
+    from typing_extensions import Literal as BPLiteral
```

Comment:
What's the "BP" for?

Reply:
BackPort?

Reply:
Yes. It can be made more explicit. I believe a different name is required later for conditional accessing the back port's literal values.

Reply:
This is okay, I was just curious.